rocketpy/mathutils/function.py

```diff
@@ -1144,6 +1144,78 @@ def low_pass_filter(self, alpha, file_path=None):
             title=self.title,
         )
 
+    def remove_outliers(self, method="iqr", **kwargs):
+        """Remove outliers from the Function source using the specified method.
+
+        Parameters
+        ----------
+        method : string, optional
+            Method to be used to remove outliers. Options are 'iqr'.
+            Default is 'iqr'.
+        **kwargs : optional
+            Keyword arguments to be passed to specific methods according to the
+            selected method.
+            If the selected method is the 'iqr', then the following kwargs are
+            available:
+            - threshold : float
+                Threshold for the interquartile range method. Default is 1.5.
+
+        Returns
+        -------
+        Function
+            The new Function object without outliers.
+        """
+        if callable(self.source):
+            print("Cannot remove outliers if the source is a callable object.")
+            return self
+
+        if method.lower() == "iqr":
+            return self.__remove_outliers_iqr(**kwargs)
+        else:
+            print(
+                f"Method '{method}' not recognized. No outliers removed."
+                + "Please use one of the following supported methods: 'iqr'."
+            )
+            return self
+
+    def __remove_outliers_iqr(self, threshold=1.5):
+        """Remove outliers from the Function source using the interquartile
```

Comment:
Here is the only place that `iqr` is mentioned in full. I believe this should be part of the documentation of the main method, not only the private one.

Reply:
You're 100% right.